app/uk/gov/hmrc/tariffclassificationfrontend/views/refer_case.scala.html

```diff
@@ -30,7 +30,7 @@
         <div class="column-full">
             <h3 class="heading-large mt-0">Refer this case</h3>
 
-            <p> Before you refer this case, <a href="#">add a note</a> explaining why you’re referring it. </p>
+            <p> Before you refer this case, <a href="@uk.gov.hmrc.tariffclassificationfrontend.controllers.routes.CaseController.activityDetails(c.reference)">add a note</a> explaining why you’re referring it. </p>
```

Comment:
is this wanted?
this is for the refer case action, not the suppress case action

Reply:
I noticed that all these "add a note" links are broken so fixed them all. Maybe it's the wrong thing to do?

Reply:
Looks fine to me